sale_order_partner_no_autofollow/models/sale_order.py

```diff
@@ -0,0 +1,45 @@
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    def message_subscribe(self, partner_ids=None, channel_ids=None, subtype_ids=None):
+        if self.env.context.get("sale_partner_disable_autofollow", False):
```

Comment:
Set default argument `partner_ids=[]`.
The condition can be reformatted:
`if self._context.get("sale_partner_disable_autofollow") and self.partner_id.id in partner_ids`

Reply:
Ok, I'll fix it)